images/build/debian-base/buster/Dockerfile

```diff
@@ -12,8 +12,86 @@
 # See the License for the specific language governing permissions and
```

Comment:
I basically removed the `Dockerfile` in favor of the `Dockerfile.build`, because the original `Dockerfile` does not seem to be used anywhere.